openslides_backend/action/actions/assignment_candidate/delete.py

```diff
@@ -17,23 +13,3 @@ class AssignmentCandidateDelete(PermissionMixin, DeleteAction):
 
     model = AssignmentCandidate()
     schema = DefaultSchema(AssignmentCandidate()).get_delete_schema()
-
-    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
-        instance = super().update_instance(instance)
-        assignment_candidate = self.datastore.get(
-            fqid_from_collection_and_id(self.model.collection, instance["id"]),
-            mapped_fields=["assignment_id"],
-        )
-        assignment = self.datastore.get(
-            fqid_from_collection_and_id(
-                "assignment", assignment_candidate["assignment_id"]
-            ),
-            mapped_fields=["phase", "meeting_id"],
-        )
-        if assignment.get("phase") == "finished" and not self.is_meeting_deleted(
-            assignment.get("meeting_id", 0)
-        ):
-            raise ActionException(
-                "It is not permitted to remove a candidate from a finished assignment!"
-            )
-        return instance
```

Comment:
Please add this back in, it should still not be permitted to delete the assignment candidate, just the user

Reply:
Re-added the check and the test.